examples/lua_REPL/Makefile.ci

```diff
@@ -1,3 +1,7 @@
+BOARD_BLACKLIST := \
+    nucleo-l011k4 \
```

Comment:
Why not `BOARD_INSUFFICIENT_MEMORY`?

Reply:
Ok, I moved this to the main Makefile with a comment (same for `lua_basic`).